evasquez26/angular4-startup
# S1-ONE 31 Remember me

## Story / Task

see [ONE-31](https://bixlabs.atlassian.net/browse/ONE-31)


## What this PR does?

Adds a functionality to remember user when loging in.


### Before 

-There's no remember me functionality

### After

- When signing in, user can select the remember me option. If selected user should remain logged in when closing browser.

### How to test / reproduce

- Login selecting the remember me option
- Close browser/tab
- Open browser tab and go to app url
- User should be logged in

### TODOs
- [x] Rebase after merging https://github.com/evasquez26/angular4-startup/pull/6

### Related PRs
N/A

## Migrations
NO


Thanks, @evasquez26
